github-electron/github-electron.d.ts

```diff
@@ -3831,12 +3853,21 @@ declare namespace Electron {
 		 * By default a new BrowserWindow will be created for the url.
 		 *
 		 * Calling event.preventDefault() will prevent creating new windows.
+		 * In such case, the event.newGuest may be set with a reference
+		 * to a BrowserWindow instance to make it used by the Electron's runtime.
 		 */
-		on(event: 'new-window', listener: (event: Event,
+		on(event: 'new-window', listener: (event: Event & { newGuest?: BrowserWindow },
```

Comment:
Type definition of `event` looks adhoc.  Should we define new type extending `Event`?

```typescript
interface NewWindowListenerEvent extends Event {
    newGuest?: BrowserWindow;
}
```